data/dbDEGEN/data.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Deribet DEGEN",
+    "symbol": "dbDEGEN",
+    "decimals": 18,
+    "description": "dbDEGEN is the official DEGEN IOU token issued by Deribet.io. The token is redeemable 1-to-1 (1 dbDEGEN = 1 DEGEN) on the platform Deribet.io",
+    "website": "https://deribet.io/",
+    "twitter": "@deribet_io",
+    "tokens": {
+        "base": {
+            "address": "0x61431b49D43220a53B4a2583870D69F67cEd8d76"
+        },
+        "nobridge": true
```

Comment:
@CaesarDB 

Thanks for the update; this should go up above `tokens`, you can put it under `twitter`. Can you please also fix the spacing and make sure the JSON validates after?

Reply:
@wbnns done - sorry for the back and forth!